File: BASIC_CODING/LARGEST_AMONG_3/solution.js
/**
 * Find and print the largest of three numbers.
 *
 * @param {number} a - The first number.
 * @param {number} b - The second number.
 * @param {number} c - The third number.
 */
function findMax(a, b, c) {
	if (a > b && a > c) {
		console.log(`The maximum number is: ${a}`);
	} else if (b > c) {
		console.log(`The maximum number is: ${b}`);
	} else {
		console.log(`The maximum number is: ${c}`);
	}
}

// Example usage:
let num1 = 3;
let num2 = 5;
let num3 = 1;

console.log(`Finding the maximum among ${num1}, ${num2}, and ${num3}:`);
findMax(num1, num2, num3);

File: BASIC_CODING/LARGEST_AMONG_3/solution.java
public class Main {
    /**
     * Find and print the largest of three numbers.
     *
     * @param a The first number.
     * @param b The second number.
     * @param c The third number.
     */
    public static void findMax(int a, int b, int c) {
        if (a > b && a > c) {
            System.out.println("The maximum number is: " + a);
        } else if (b > c) {
            System.out.println("The maximum number is: " + b);
        } else {
            System.out.println("The maximum number is: " + c);
        }
    }

    public static void main(String[] args) {
        int num1 = 3;
        int num2 = 5;
        int num3 = 1;

        System.out.println("Finding the maximum among " + num1 + ", " + num2 + ", and " + num3 + ":");
        findMax(num1, num2, num3);
    }
}
